doc/contributing/development-environment.md

```diff
@@ -28,46 +39,41 @@ This will give you the `master` branch of TerriaJS.  While we strive to keep `ma
 ```
 grep terriajs package.json
 # will print something like: "terriajs": "^4.5.0"
-cd node_modules/terriajs
+cd packages/terriajs
 git checkout 4.5.0
-cd ../..
+cd ..
 ```
 
-Now, if you run `npm install`, `npm` will fail with an error like this:
-
-```
-npm ERR! git C:\github\TerriaMap\node_modules\terriajs: Appears to be a git repo or submodule.
-npm ERR! git     C:\github\TerriaMap\node_modules\terriajs
-npm ERR! git Refusing to remove it. Update manually,
-npm ERR! git or move it out of the way first.
-```
+!!! note
+    
+    The version of TerriaJS in your `packages/terriajs` directory must be semver-compatible with the version specification in TerriaMap's `package.json`. If it's not, yarn will install a separate semver-compatible version of TerriaJS in `node_modules` instead of using the one you've put in `packages/terriajs`. The version numbers are _usually_ already aligned, but if not, change the `"terriajs": "x.y.z"` dependency in TerriaMap's `package.json` to be the exact `"version"` property in TerriaJS's `package.json`. You will generally not want to commit this change.
```

Comment:
For some reason that line is displayed as code and is not wrapping
<img width="901" alt="screen shot 2017-12-05 at 12 09 20 pm" src="https://user-images.githubusercontent.com/7508939/33584794-25c85c6a-d9b5-11e7-800b-dc2b681ec606.png">

Reply:
It should get formatted correctly on the doc site.  GitHub's markdown rendering doesn't have the necessary plugin.